packages/store/src/migrations/InitialSetup20240227075933.ts

```diff
@@ -0,0 +1,48 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class InitialSetup20240226130829 extends Migration {
+  async up(): Promise<void> {
+    // TODO: The migration needs to take care of moving data before dropping columns, among other things
+    const storeTables = await this.execute(
```

Comment:
question: I am seeing the following error in module integration tests (using the module test runner) following the same approach:
```
relation "information_schema.tables" does not exist
```

Did you encounter this? It looks as if the database is not set up properly. We might need a try catch here.